sbt-crossproject/src/main/scala/sbtcrossproject/CrossType.scala

```diff
@@ -37,6 +37,13 @@ abstract class CrossType {
    */
   def sharedSrcDir(projectBase: File, conf: String): Option[File]
 
+  /** The location of a shared resources directory (if it exists)
+   *  @param projectBase the base directory of a (true sbt) Project
+   *  @param conf name of sub-directory for the configuration (typically "main"
+   *      or "test")
+   */
+  def sharedResourcesDir(projectBase: File, conf: String): Option[File]
```

Comment:
Hum, we cannot add an abstract method. That is not backward binary compatible, as it will break existing subclasses in the wild that do not implement that method. Normally the CI would catch that, but we had not set up MiMa yet. I submitted a PR #134 to set it up.

Instead, this method should be concrete, with a default implementation (probably `= None`).

Reply:
Also, since this adds a public method, it will require a minor version bump. So you should first include a commit (in this PR) that bumps the next version to 1.1.0-SNAPSHOT (in `project/Extra.scala`), and then a second commit with these changes.

Reply:
Oops, right. Fixed. Do you want me to rebase against #134 / master once its merged?

Reply:
Yes, ideally we should wait for #134 to be merged and then rebase this one on top afterwards.